shadered-remove-sdl-error-check.patch

```diff
@@ -0,0 +1,18 @@
+--- shadered-1/main.cpp.remove-sdl-error-check	2020-02-29 00:00:29.306169682 +0100
++++ shadered-1/main.cpp	2020-03-01 22:57:15.473286078 +0100
+@@ -136,12 +136,9 @@
+
+
+ 	// init sdl2
+-	if (SDL_Init(SDL_INIT_VIDEO | SDL_INIT_TIMER | SDL_INIT_AUDIO) < 0) {
```

Comment:
I still think we need to figure out why this is happening. There might be something wrong with the runtime.

Reply:
My guess is SDL actually can't initialize audio (since it has no means to play sound) and thus returns error. `--socket=pulseaudio` should help.

Reply:
Indeed, it makes sense for SHADERed to use PulseAudio as it features audio visualization support.